### ISSUE TITLE
Adjust contrast of buttons on homepage/three-button-call-to-action.html

### ISSUE THREAD

**Issue by eugenethreat** [phillycommunitywireless/phillycommunitywireless#242]
The colors we chose for the center and right button in `three-button-call-to-action.html` fail accessibility standards...should be adjusted. 

<img width="1746" height="944" alt="Image" src="https://github.com/user-attachments/assets/aa6a2b33-bc4b-4f91-ab90-a94c5f9d2f76" />

<img width="1294" height="641" alt="Image" src="https://github.com/user-attachments/assets/6ff9dc45-0853-4e38-a7ac-ef19e89ceae7" />

**Comment by leannepcw** [comment]
@eugenethreat the pinkish colour is from our branding palette and i don't think the colour itself should be changed. we could instead make all the buttons the primary blue-purple colour?

**Comment by hawc2** [comment]
fixed by #259
